### ISSUE TITLE
API Access - Pending requests

### ISSUE THREAD

**Issue by ATCADS** [OfficeDev/microsoft-teams-emergency-operations-center#75]
After following the deployment guide and working through step 8: Deploy "NotifyToTeams" we are not seeing any pending requests show up for sub-step 11.  Not sure if this is also affecting the app within Teams showing a blank page below the header.

![image](https://user-images.githubusercontent.com/77405614/222225464-3cd48424-d3ba-43ca-a501-4f623c04868e.png)

**Comment by v-prigunasek** [comment]
Hi @ATCADS,

Thanks for reaching out to us. The API permissions are for the EOC-Extension and it is a separate component so that will not affect the TEOC application. Could you please open the TEOC application in browser version of MS Teams and share me the console logs?

To get a console logs, please try to access the application in the browser version of MS Teams and open Developer tools from browser (or press Ctrl + Shift + I , this will open the console window) and then right click on the logs to Save it as a log file.  

![image](https://user-images.githubusercontent.com/98514691/222342029-41338e5a-e38c-49e9-8f1f-a89190b9a3d6.png)

![image](https://user-images.githubusercontent.com/98514691/222342220-b106eced-7737-4435-97e0-b346399ef02c.png)

For the issue related to the API request, can you please share me the screenshot of the AppCatalog library from SharePoint site and the API access page from SharePoint admin portal?

**Comment by ATCADS** [comment]
Screenshot of API Access. I did notice that both permissions for Microsoft Graph did show without requiring approval, which is confusing as they never showed in Pending requests...
<img width="887" alt="APIAccess" src="https://user-images.githubusercontent.com/77405614/222469724-f6093892-5af5-4d94-97d7-b57b0a86132f.png">

**Comment by ATCADS** [comment]
Let me rephrase, two of the nine permissions required we allowed; however, the other seven have not shown in pending yet.

**Comment by v-prigunasek** [comment]
> Screenshot of API Access. I did notice that both permissions for Microsoft Graph did show without requiring approval, which is confusing as they never showed in Pending requests... <img alt="APIAccess" width="887" src="https://user-images.githubusercontent.com/77405614/222469724-f6093892-5af5-4d94-97d7-b57b0a86132f.png">

You are good with the API permissions, since those 2 permissions are already listed under approved requests. That is the reason you were not seeing it under Pending requests.

Thanks!!

**Comment by ATCADS** [comment]
Setting the site in the Application setting did the trick! Thank you so much!
<img width="1105" alt="TeamsEOC" src="https://user-images.githubusercontent.com/77405614/222476889-df720fd4-4335-4eb8-8f85-6c0bfae062ae.png">

**Comment by v-prigunasek** [comment]
That sounds perfect!! Thanks for your interest in using TEOC Application. Please feel free to reach out to us for any queries/issues.
